### ISSUE TITLE
`samtools` compatibility problem: sorting

### ISSUE THREAD

**Issue by widdowquinn** [widdowquinn/THAPBI-pycits#69]
The `samtools` wrapper fails with

```
$ samtools --version
samtools 1.3
Using htslib 1.3.1
Copyright (C) 2015 Genome Research Ltd.
```

as with this version, the `-o` flag is necessary to indicate the output directory.

`nosetests -v` reports:

```
samtools_sort instantiates, runs and returns ... ERROR
Run samtools_index on test data and compare output ... ERROR
```

**Comment by widdowquinn** [comment]
The following fixes the issue:

```diff
diff --git a/pycits/samtools_sort.py b/pycits/samtools_sort.py
index 805602e..f46f8a2 100644
--- a/pycits/samtools_sort.py
+++ b/pycits/samtools_sort.py
@@ -55,5 +55,5 @@ class Samtools_Sort(object):
                "-@",
                threads,
                infname,
-               self._outfname]
+               '-o', self._outfname]^M
         self._cmd = ' '.join(cmd)
```

and still works with `samtools 1.4`.

**Comment by widdowquinn** [comment]
Fixed with c4cb35a
